plugins/core/IO/qLASIO/src/LasDetails.cpp

```diff
@@ -268,11 +268,8 @@ namespace LasDetails
 			if (hasWaveform || previousMinorVersion >= 3)
 			{
 				minorVersion = 3;
-				if (hasGpsTime)
```

Comment:
Wouldn't the test be instead:
```
if (hasWaveform || hasGpsTime)
{
    pointFormat = 4;
}
```

Reply:
Not sure of that. As soon as you have waveforms, to should get either 4 or 5 as a point format, GpsTime being in both.
Maybe the test
`if (hasWaveform || previousMinorVersion >= 3) `
should besimply  changed to
`if (hasWaveform)`
as minorVersion is set to 3 after the test

Reply:
@tmontaigu any preference?

Reply:
I think it should be
```c++
int  minorVersion = previousMinorVersion == 3 ? 3 : 2;
if (hasWaveform) {
   pointFormat = 4;
   // ...
}
```

Reply:
OK, let's do it that way. I will change the test.